test/benchmark_utils/callgrind_artifacts.json

```diff
@@ -0,0 +1,1187 @@
+{
```

Comment:
Are you sure you actually want to check this in? Need to keep reading to find out why this is used, but this seems a bit fragile.

Reply:
It's to hermetically mock a collected Callgrind so some of the supporting functionality can be tested as a fast unit test.

Reply:
I see now that this is a test fixture.

Reply:
It would have been nice if this fixture were smaller. Not sure if this is actually possible, most of the lines are coming from Python.